test/e2e/ci.sh

```diff
@@ -24,3 +24,13 @@ create_cluster ${KIND_K8S_VERSION}
 trap delete_cluster EXIT
 
 test_cluster
+
+KUSTOMIZATION_FILE_PATH="$(dirname "$0")/../../testdata/project-v4/config/default/kustomization.yaml"
+
+sed -i '25s/^#//' $KUSTOMIZATION_FILE_PATH
+sed -i '27s/^#//' $KUSTOMIZATION_FILE_PATH
+sed -i '42s/^#//' $KUSTOMIZATION_FILE_PATH
+sed -i '46,143s/^#//' $KUSTOMIZATION_FILE_PATH
+
+cd $(dirname "$0")/../../testdata/project-v4
+make test-e2e
```

Comment:
This one is called to do the prow tests
I think we need a new configuration to perform this specific tests
Ideally, use GitHub Actions instead. 
So that would result in a new GitHub action that will setup the kind and run the tests scaffold for the sample